enginetest/enginetests.go

```diff
@@ -1786,6 +1786,170 @@ func TestCreateDatabase(t *testing.T, harness Harness) {
 	})
 }
 
+func TestPkOrdinals(t *testing.T, harness Harness) {
+	e := NewEngine(t, harness)
+	ctx := NewContext(harness)
+
+	TestQuery(t, harness, e, "CREATE DATABASE testdb", []sql.Row{{sql.OkResult{RowsAffected: 1}}}, nil, nil)
```

Comment:
Probably don't want to create DBs like this if you can avoid it, just use myDb like other tests

Reply:
👍